package-lock.json

```diff
@@ -0,0 +1,4420 @@
+{
```

Comment:
Yea I've been removing it during commits, just forgot to remove this one. I'm pretty sure it's harmless though (much like `yarn.lock`).